notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -69,9 +69,9 @@
    "outputs": [],
    "source": [
     "%pip uninstall -q -y openvino-dev openvino openvino-nightly\n",
-    "%pip install -q \"openvino-nightly\" \"nncf>=2.7\"\n",
-    "%pip install -q datasets \"torchmetrics>=0.11.0\" --extra-index-url https://download.pytorch.org/whl/cpu\n",
-    "%pip install -q soundfile librosa transformers --extra-index-url https://download.pytorch.org/whl/cpu"
+    "%pip install -q openvino-nightly \"nncf>=2.7\"\n",
+    "%pip install datasets \"torchmetrics>=0.11.0\" \"torch>=2.1.0\" --extra-index-url https://download.pytorch.org/whl/cpu\n",
+    "%pip install -q soundfile librosa \"transformers>=4.36.2\""
```

Comment:
transformers also may have requirements on specific torch version and reinstall it, so better to stay --extra-index-url on this line too for avoid reinstalling torch with cuda

Reply:
Added